docs/debugging.md

```diff
@@ -73,22 +77,33 @@ You may use commands such as `crane`, `oras`, `crictl` or `docker` to verify the
 2023/03/17 04:45:49 --> GET https://registry.k8s.io/v2/pause/blobs/sha256:61fec91190a0bab34406027bbec43d562218df6e80d22d4735029756f23c7007 [body redacted: omitting binary blobs from logs]
 2023/03/17 04:45:49 --> GET https://prod-registry-k8s-io-us-west-2.s3.dualstack.us-west-2.amazonaws.com/containers/images/sha256%3A61fec91190a0bab34406027bbec43d562218df6e80d22d4735029756f23c7007 [body redacted: omitting binary blobs from logs]
 ```
+
 From my location, the pull command accesses registry.k8s.io, us-west1-docker.pkg.dev and prod-registry-k8s-io-us-west-2.s3.dualstack.us-west-2.amazonaws.com. You will need to have DNS and HTTP access to these domains on your node to pull images.
 
+It'a also possible to run these commands on your node if you don't have SSH access by using `kubectl run`:
+
+```sh
+kubectl run --rm -it crane --restart=Never --image=gcr.io/go-containerregistry/crane --overrides='{"spec": {"hostNetwork":true}}' -- pull --verbose registry.k8s.io/pause:3.9 pause.tgz
```

Comment:
this one doesn't need to be host network because we're pulling from registry.k8s.io/pause in the node runtime anyhow. it's a good option to provide though

I think for at least `crane pull --verbose registry.k8s.io/pause` we should provide an example to run it in-cluster with host network so we can see the pull logs (versus the above test which will be easier to run without any privileges but may only turn up failure or success and any image pulling logs will be in the node's container runtime)

Reply:
i'm an uncaffeinated idiot.